robot/Cumulus/resources/NPSP.robot

```diff
@@ -51,6 +51,22 @@ API Modify Contact
     &{contact} =  Get From List  ${records}  0
     [return]         &{contact}
 
+API Modify Recurring Donation
```

Comment:
Since you have added a new keyword, can you regenerate the libdoc task once ?

Reply:
Thanks for reminding me @Safiyab